From 6e78d02a822848205d1622748f3d7474f802a48e Mon Sep 17 00:00:00 2001
From: Maksym Glubokiy <maksym.glubokiy@plvision.eu>
Date: Tue, 25 Jan 2022 15:06:08 +0200
Subject: [PATCH 36/40] Revert "armada-70x0: increase PCIe memory size"

Latest AC5x devices request PCI memory region which exceeds the one
defined in generic DT (armada-70x0.dtsi).
At the same time older devices are fine with the original PCI ranges.

Restore original ranges from upstream kernel (this commit)
and increase them in platform-specific DTS(s) for platforms
that require it (separate commit).

This reverts commit 186d0022d0f5de3c1be00ace2f050539b6cd3e84.

JIRA: SWITCHDEV-2685

Signed-off-by: Maksym Glubokiy <maksym.glubokiy@plvision.eu>
Change-Id: Id78fc1731e1ddc5fc3f509ba767f5feb24868aa1
---
 arch/arm64/boot/dts/marvell/armada-70x0.dtsi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/marvell/armada-70x0.dtsi b/arch/arm64/boot/dts/marvell/armada-70x0.dtsi
index 3f8184a471b3..293403a1a333 100644
--- a/arch/arm64/boot/dts/marvell/armada-70x0.dtsi
+++ b/arch/arm64/boot/dts/marvell/armada-70x0.dtsi
@@ -20,7 +20,7 @@
 #define CP11X_NAME		cp0
 #define CP11X_BASE		f2000000
 #define CP11X_PCIEx_MEM_BASE(iface) (0xf6000000 + (iface * 0x1000000))
-#define CP11X_PCIEx_MEM_SIZE(iface) 0x1800000
+#define CP11X_PCIEx_MEM_SIZE(iface) 0xf00000
 #define CP11X_PCIE0_BASE	f2600000
 #define CP11X_PCIE1_BASE	f2620000
 #define CP11X_PCIE2_BASE	f2640000
-- 
2.17.1

